.circleci/config.yml

```
@@ -1,4 +1,4 @@
-version: "2.1"
+version: 2.1
```

[Here](https://circleci.com/docs/configuration-reference/#version) it says that `version` should be a string. So better change this back to

```suggestion
version: "2.1"
```

---

Schemas :D Somehow my vim got the schema for it and it says exactly the opposite!